packages/roosterjs-content-model-plugins/lib/paste/PastePlugin.ts

```diff
@@ -219,3 +224,15 @@ function tableBorderParser(format: ContentModelTableCellFormat, element: HTMLEle
         }
     });
 }
+
+/**
+ * @internal exported only for unit test
+ */
+export const pasteButtonProcessor: ElementProcessor<HTMLButtonElement> = (
+    group: ContentModelBlockGroup,
+    element: HTMLButtonElement,
+    context: DomToModelContext
+): void => {
+    const text = createText(element.textContent || '', context.segmentFormat);
```

Comment:
I think we should keep the format of the text on button. So if it has some special format, user can easily distinguish them

Reply:
Sounds good, looks like button can contain blocks and segments but still be inline, so we need to check all child elements. 
Addressed this in latest commit